lib/ansible/plugins/connection/ssh.py

```diff
@@ -558,10 +558,14 @@ def _add_args(self, b_command, b_args, explanation):
         display.vvvvv(u'SSH: %s: (%s)' % (explanation, ')('.join(to_text(a) for a in b_args)), host=self._play_context.remote_addr)
         b_command += b_args
 
-    def _build_command(self, binary, *other_args):
+    def _build_command(self, binary, subsystem, *other_args):
         '''
-        Takes a binary (ssh, scp, sftp) and optional extra arguments and returns
-        a command line as an array that can be passed to subprocess.Popen.
+        Takes a executable (ssh, scp, sftp or wrapper) and optional extra arguments and returns from remote device.
+
+        :binary: actual executable to use to execute command.
+        :subsystem: type of executable provided, ssh/sftp/scp, needed because wrappers for ssh might have diff names.
+        :other_args: dict of, value pairs passed as arguments to the ssh binary
```

Comment:
This inline doc formatting is odd to me. Are you trying to do sphinx format maybe?

```
:param binary: ...
:param subsystem: ...
:param other_args: ...
```